source/getting-started/onboarding_dashboard.markdown

```diff
@@ -3,11 +3,21 @@ title: "Edit the dashboard"
 description: "Instructions on editing the dashboard for the first time"
 ---
 
+By default, Home Assistant comes with [multiple dashboards](/dashboards/dashboards/):
+
+- Overview
+- Energy
+- Map
+- Logbook
+- History
+
+In this tutorial, we are going to edit the **Overview** dashboard to get familiar with some basic dashboard concepts.
+
 ## First contact with the Overview dashboard
 
 The **Overview** [dashboard](/dashboards/) is the first page you see after the [onboarding process](/getting-started/onboarding). Dashboards are customizable pages to display information in Home Assistant.
 
-By default, there are two dashboards: **Overview** and **Energy**. The image below shows a customized example of the **Overview** dashboard. If you just onboarded, your dashboard will be nearly empty.
+The image below shows a customized example of the **Overview** dashboard. If you just onboarded, your dashboard will be nearly empty.
```

Comment:
Honestly, I think that approach is not good.

Basically, we tell the user to take control of their default generated overview dashboard when getting started. As a result, their dashboard will not automatically update anymore when they continue their journey.